### PR TITLE
Add optimus adapter

### PR DESCRIPTION
**Pull request by robertdowneyjr0113**
##### Short Description (to be shown on DefiLlama): Optimus is an on-chain yield optimizer that maximizes your investment return with a minimal risk level possible.
Our vision is to enable everyone in the world to effortlessly invest in and profit from DeFi (Decentralized Finance) protocols built on the ICON Network. Our very first mission is to develop a lending aggregator, which seeks to optimize investment opportunities in the existing ICON ecosystem provided by lending pools such as Balanced and Omm.

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/optimus exports TVL: 
        
 
 
    
    total                    4.20 M

**Comment by waynebruce0x** [comment]
Are borrowed funds included in the count?

File: projects/optimus/index.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+      ommSIcxRate(),
+      getOMMLendingStatus(),
+      getOMMPrice(),
+      getOmmAutoStakingStatus(),
+  ]);
+
+  const values = await Promise.all([
+    totalSupply.times(ICXPrice),
+    loanInfo.times(ICXPrice),
+    fTokenPool.times(fTokenRate).times(ommRatesIcx).times(ICXPrice),
```
[the remark this answers is not in the document]
Comment (robertdowneyjr0113) [reply projects/optimus/index.js:119]: We dont display `fTokenPool` value in UI, but we have total supply in ICX at here:
image

**Comment by robertdowneyjr0113** [comment]
> Are borrowed funds included in the count?

Borrowing funds (bnUSD) in Borrower strategy are not included in TVL, TVL on Optimus includes the total supply deposited into 4 strategies (LP - Balanced, Borrower - Balanced, Lender - OMM and Staker - OMM)

> Do you mind giving a bit more of a detailed explanation to what is being counted and where it can be accessed on the UI?

You can access https://optimus.finance and you see TVL display at HomePage
image

>  Is the API backend code open source?

No, We don't publish backend source code now.

File: projects/optimus/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (robertdowneyjr0113) [reply projects/optimus/index.js:119]: fToken you say is `fTokenPool` in this picture

**Pull request merged**